axlearn/common/trainer_config_modifier.py

```diff
@@ -0,0 +1,180 @@
+# Copyright © 2023 Apple Inc.
```

Comment:
```suggestion
# Copyright © 2024 Apple Inc.
```
(here and elsewhere)

Reply:
Done. Thanks for the review, I need one more approval after the nit change.